docs/reference/builder.md

```diff
@@ -379,9 +379,13 @@ the working and the root directory.  For example, the patterns
 in the `foo` subdirectory of `PATH` or in the root of the git
 repository located at `URL`.  Neither excludes anything else.
 
+If a line in `.dockerignore` file starts with `#`, then this line is
+considered as a comment and is ignored when interpreted by the CLI.
+
 Here is an example `.dockerignore` file:
 
 ```
+    # comment
```

Comment:
We should remove the leading spaces from the example, it is confusing, even though previously valid.